migrations/migration_encrypt_shares.go

```diff
@@ -0,0 +1,46 @@
+package migrations
+
+import (
+	"context"
+	"crypto/sha256"
+	"crypto/x509"
+	"fmt"
+
+	"go.uber.org/zap"
+)
+
+var encryptSharesMigration = Migration{
```

Comment:
```suggestion
var migrationEncryptShares = Migration{
```

just to keep the naming convention of other migrations

this is just so that IDEs would autocomplete nicely when you start typing "migration"